ambry-store/src/test/java/com.github.ambry.store/StoreDescriptorTest.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.store;
+
+import java.io.DataOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.UUID;
+import junit.framework.Assert;
+import org.junit.Test;
+
+
+/**
+ * Tests {@link StoreDescriptor}
+ */
+public class StoreDescriptorTest {
+
+  /**
+   * Tests {@link StoreDescriptor} for unit tests for instantiation and converting bytes into StoreDescriptor
+   * @throws IOException
+   */
+  @Test
+  public void testStoreDescriptor() throws IOException {
+    String tempDirPath = "/tmp/";
+    File storeDescriptorFile = new File(tempDirPath + "/" + StoreDescriptor.STORE_DESCRIPTOR_FILENAME);
+    storeDescriptorFile.delete();
```

Comment:
if you use `StoreTestUtils::createTempDirectory()`, this will no longer be required.